test/e2e/framework/pod/exec_util.go

```diff
@@ -68,8 +68,7 @@ func ExecWithOptionsContext(ctx context.Context, f *framework.Framework, options
 		Resource("pods").
 		Name(options.PodName).
 		Namespace(options.Namespace).
-		SubResource("exec").
-		Param("container", options.ContainerName)
```

Comment:
I fixed this while working on https://github.com/deads2k/kubernetes/pull/10: the container parameter in the URL gets added through v1.PodExecOptions and shouldn't be here (IMHO).

Might be better in a separate PR because it's not directly related. But it's small, so perhaps not worth it?